packages/workers/src/scripts/telegram-mixpanel.ts

```diff
@@ -0,0 +1,41 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import Mixpanel from 'mixpanel';
+let mixpanel: Mixpanel.Mixpanel;
+
+import TelegramBot from 'node-telegram-bot-api';
+let sommBot: TelegramBot | undefined;
+
+
+
+if(process.env.SOMM_STATS_BOT_TOKEN){
```

Comment:
Please share secret

Also, how often should this run? Can you start updating `workers/src/scheduler.ts` with a properly-scheduled cronjob for this script?